sim/battle.js

```diff
@@ -1795,7 +1793,18 @@ class Battle extends Dex.ModdedDex {
 		defense = this.runEvent('Modify' + statTable[defenseStat], defender, attacker, move, defense);
 
 		//int(int(int(2 * L / 5 + 2) * A * P / D) / 50);
-		let baseDamage = Math.floor(Math.floor(Math.floor(2 * level / 5 + 2) * basePower * attack / defense) / 50) + 2;
+		let baseDamage = Math.floor(Math.floor(Math.floor(2 * level / 5 + 2) * basePower * attack / defense) / 50);
+		
+		// Calculate initial damage modifiers (order differs between generations)
+		baseDamage = this.modifyDamage(pokemon, target, move, basePower, baseDamage, suppressMessages);
```

Comment:
Hm...I passed this because it was in the original code so I figured there was something I was missing, but I've gone through each event and there's nothing that can modify basePower after ``if (!basePower) return 0;`` so I guess it's fine to assume it isn't 0. Makes things prettier anyway. Code has been updated.